### PR TITLE
Print repeated Problem for Problem page

### PR DESCRIPTION
**Pull request by lalaAdasia00**

### CODE REVIEW

File: problems/printRepeated.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+tests:
+  - assert(printRepeated('hello', 4, 3) === 'hellohellohellohello\nhellohellohellohello\nhellohellohellohello');
+  - assert(printRepeated('YellowBrickCode', 2, 4) === 'YellowBrickCodeYellowBrickCode\nYellowBrickCodeYellowBrickCode\nYellowBrickCodeYellowBrickCode\nYellowBrickCodeYellowBrickCode');
+solution:
+  function repeatString(str, repeatNum){
+    var newStr = '';
+
+    for(var i = 0; i < repeatNum; i++){
+        newStr += str;
+    }
```
Comment (nicholashusin) [inline problems/printRepeated.yaml]: Missing `return` statement

**Review by nicholashusin — CHANGES_REQUESTED** [review]

File: problems/printRepeated.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  'hellohellohellohello'
+  'hellohellohellohello'
+  'hellohellohellohello'
+code: |
+  function printRepeated(str, a, b) {
+    return str;
+  }
+
+
+  /* Hint- Helper fucntion may be used */
```
Comment (ifeomi) [inline problems/printRepeated.yaml]:
```suggestion
  /* Hint- Helper function may be used */
```

File: problems/printRepeated.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+id: 5
+title: PrintRepeated
+text: |
+  Write the function called printRepeated that, given a string str, an integer a, 
+  and an integer named b, prints out the String str b times, 
+  where each string is repeated a times.
+  
+  Refer to example below.
+  
+  printRepeated('hello', 4, 3) =
```
Comment (ifeomi) [inline problems/printRepeated.yaml]: should we change the name to generateRepeated?

File: problems/printRepeated.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+id: 5
+title: PrintRepeated
+text: |
+  Write the function called printRepeated that, given a string str, an integer a, 
+  and an integer named b, prints out the String str b times, 
+  where each string is repeated a times.
```
Comment (ifeomi) [inline problems/printRepeated.yaml]: should rephrase since they aren't printing

**Review by chronologos — APPROVED** [review]

**Review by nicholashusin — APPROVED** [review]

**Pull request merged**